tests/unit/tuf_repository_service_worker/models/targets/test_crud.py

```diff
@@ -8,35 +8,65 @@
 from repository_service_tuf_worker.models.targets import crud
 
 
-class TestTargetsCrud:
-    def test_create(self):
-        crud.models.RSTUFTargets = pretend.call_recorder(
-            lambda *a, **kw: "fake_db_target"
+class TestCrud:
+    def test_create_roles(self, monkeypatch):
+        monkeypatch.setattr(
+            crud.models,
+            "RSTUFTargetRoles",
+            pretend.call_recorder(lambda *a, **kw: "test_target_roles"),
+        )
+        mocked_db = pretend.stub(
+            add_all=pretend.call_recorder(lambda *a: None),
+            commit=pretend.call_recorder(lambda: None),
+        )
+        test_target = crud.schemas.RSTUFTargetRoleCreate(
+            rolename="bins-0",
+            version=1,
+        )
+        test_result = crud.create_roles(mocked_db, [test_target])
+        assert test_result == ["test_target_roles"]
+        assert mocked_db.add_all.calls == [pretend.call(["test_target_roles"])]
+        assert mocked_db.commit.calls == [pretend.call()]
+
+    def test_create_file(self, monkeypatch):
+        monkeypatch.setattr(
+            crud.models,
+            "RSTUFTargetRoles",
+            pretend.call_recorder(lambda *a, **kw: "test_target_roles"),
+        )
+        monkeypatch.setattr(
+            crud.models,
+            "RSTUFTargetFiles",
+            pretend.call_recorder(lambda *a, **kw: "test_target_files"),
         )
         mocked_db = pretend.stub(
             add=pretend.call_recorder(lambda *a: None),
             commit=pretend.call_recorder(lambda: None),
             refresh=pretend.call_recorder(lambda *a: None),
         )
-        test_target = crud.schemas.TargetsCreate(
+        test_target_file = crud.schemas.RSTUFTargetFileCreate(
             path="file1.tar.gz",
             info={"info": {"k": "v"}},
-            rolename="bins-0",
             published=False,
             action=crud.schemas.TargetAction.ADD,
         )
```

Comment:
Because this object is used to create the RSTUFTargetFile as a call in the `create_file` https://github.com/repository-service-tuf/repository-service-tuf-worker/blob/14fef8bbdb32cf348f1251bd2a2007f6836844fc/repository_service_tuf_worker/models/targets/crud.py#L36